ch07.adoc

```diff
@@ -544,3 +544,88 @@ data:  // time coordinates translated to date/time format
                      "2000-8-1 6:00:00", "2000-9-1 6:00:00" ;
 ----
 ====
+
+[[spatial-geometries, Section 7.5, "Spatial Geometries"]]
+=== Spatial Geometries
+
+For many geospatial applications, data values are associated with a spatial geometry (e.g., the average monthly rainfall in the UK).  Although cells with an arbitrary number of multiple vertices can be described using <<cell-boundaries>>, spatial geometries contain an arbitrary number of nodes for each geometry and include line and __multipart__ geometries (e.g., the different islands of the UK).  The approach described here specifies how to encode such geometries following the pattern in **9.3.3 Contiguous ragged array representation** and attach them to variables in a way that is consistent with the cell bounds approach.
```

Comment:
If you have a comment on a small substring of a paragraph it could be done like:

> Although cells with an arbitrary number of multiple vertices can be described using  

to  

Although cells with an arbitrary number of vertices can be described using

Or, just copy the whole paragraph and rewrite as needed.

Or, this could just be a general comment indicating how you think this should be rewritten.